### ISSUE TITLE
Working as server but not working as client

### ISSUE THREAD

**Issue by ghost** [input-leap/input-leap#737]
> This issue has been migrated from old Barrier Github repository https://github.com/debauchee/barrier/issues/737
> 
> Issue created on: 2020-06-04 by @mcnaveen
> Issue last updated on: 2020-06-04

### Operating Systems ###

Server: Ubuntu 20.04

Client: Windows 7

### Barrier Version ###

2.3.2

### Steps to reproduce bug ###

Started the Server. Checked logs...

```
[2020-06-04T22:50:17] INFO: starting server
[2020-06-04T22:50:17] INFO: config file: /tmp/Barrier.f35726
[2020-06-04T22:50:17] INFO: log level: INFO
[2020-06-04T22:50:17] NOTE: started server (IPv4/IPv6), waiting for clients

```

In the Client Log...

```
[2020-06-04T23:06:10] NOTE: connecting to '192.168.31.201 , 192.168. 31.201:24800
2020-06-04T23:06:101 INFO : Openssl 1.0.21 25 May 2017
[2020-06-04T23:06:25] WARNING: failed to connect to server: Timed out
```

### Other info ###

I tried to swap the client as a server and server as a client.
It works this way.

But, Previously I used like this.

While Upgrading from Ubuntu 18.04 to 20.04 I did the fresh installation. After that, this problem occurs.

---

> Commented on: 2020-06-04 by @mcnaveen

Running this command on Ubuntu (Barrier Server) Solved the issue.

`sudo iptables -I INPUT -p tcp --dport 24800 --syn -j ACCEPT`

Basically What it does is opening the TCP Port 24800 for the Barrier server.

---

> Closed on: 2020-06-04 by @mcnaveen
